ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml

```diff
@@ -164,6 +183,25 @@ keys:
           bfd:
             type: bool
             default: False
+          bfd_timers:
+            type: dict
+            description: |-
+                Specify bfd timer values to override the default values.
+                It is recommended to keep bfd timer higher than dps timer.
+                Default BFD timer is 6 secs as 5 secs is the default DPS timer.
+            documentation_options:
+              hide_keys: true
+            $ref: "eos_cli_config_gen#/keys/router_bgp/keys/peer_groups/items/keys/bfd_timers"
+            keys:
+              interval:
+                type: int
+                default: 2000
+              min_rx:
+                type: int
+                default: 2000
+              multiplier:
+                type: int
+                default: 3
```

Comment:
```suggestion
              interval:
                type: int
                default: 1000
              min_rx:
                type: int
                default: 1000
              multiplier:
                type: int
                default: 10
```

Reply:
Are we sure on these defaults?
The default I used is what I fetched from some previous PM discussions.